README.md

```diff
@@ -5,6 +5,7 @@ LevelUP
 
 > Fast and simple storage. A node.js wrapper for `abstract-leveldown` compliant backends.
 
+![Node version](https://img.shields.io/badge/Node.js-%3E%3D6.0.0-orange.svg?style=flat-square)
```

Comment:
With `engines.node` set, you can use `![Node version](https://img.shields.io/node/v/levelup.svg)` (badge becomes available after npm publish)

Reply:
Aha, didn't know this. I was wondering why `leveldown` had a "n/a" badge.

Reply:
This is sort of bad for native modules (here is ok I guess) since we don't want to publish a new `leveldown` version just to get a working badge in the readme.

Reply:
Hmm yeah, so let's switch 'em :) have a hardcoded badge in leveldown until the next release

Reply:
Fixed.